docs/root/start/sandboxes/opentelemetry.rst

```diff
@@ -112,6 +113,81 @@ In the ``Latency Samples`` of ``opentelemetry.proto.collector.trace.v1.TraceServ
 
 .. image:: /start/sandboxes/_static/opentelemetry-ui-traces.png
 
+Step 5: View the access logs in the OpenTelemetry collector output
+******************************************************************
+
+Check the spans Docker-compose logs:
```

Comment:
```suggestion
Check the spans Docker Compose logs:
```

i think we want to be explicit about how to do this - i would add as the first line in the console logs below something like:

```console
$ docker compose logs opentelemetry
```

Reply:
Done in b4a96fe0f5875d108a49b54fa30720273c35dcb8